server/api/reports.py

```diff
@@ -697,8 +697,8 @@ def sampled_ballot_rows(election: Election, jurisdiction: Jurisdiction = None):
             result_columns.append(f"Audit Result: {contest.name}")
             if show_cvrs:
                 result_columns.append(f"CVR Result: {contest.name}")
-                result_columns.append(f"Vote Delta: {contest.name}")
-                result_columns.append(f"Discrepancy: {contest.name}")
+                result_columns.append(f"Change in Results: {contest.name}")
+                result_columns.append(f"Change in Margin: {contest.name}")
```

Comment:
I like the new column names!

Likely a relic of an earlier iteration, but the PR description says that "Vote Delta" was changed to "Tally Discrepancies" when it looks like it was actually changed to "Change in Results" - might want to update the PR description accordingly (I like "Change in Results")

Reply:
Yes, thank you! I remember writing that and thinking it might become outdated lol